UITests/TopTabsTests.swift

```diff
@@ -23,28 +23,32 @@ class TopTabsTests: KIFTestCase {
     }
     
     private func testAddTab() {
+        tester().waitForTappableViewWithAccessibilityLabel("Show Tabs", value: "1", traits: UIAccessibilityTraitButton)
+        
         tester().tapViewWithAccessibilityIdentifier("url")
         
-        for i in 0...numberOfTabs {
+        for i in 0...(numberOfTabs-1) {
```

Comment:
Nit: You can also use the `..<` operator instead of `...` which will do the -1 for you.